frontend/apps/web/app/(mgmt)/[account]/new/job/connect/page.tsx

```diff
@@ -341,172 +412,252 @@ export default function Page({ searchParams }: PageProps): ReactElement {
+                                              shouldTouch: true,
+                                              shouldValidate: true,
+                                            }
+                                          );
                                         }
-                                      />
-                                    </SelectContent>
-                                  </Select>
+                                      }}
+                                      value={field.value}
+                                    >
+                                      <SelectTrigger
+                                        className={cn(
+                                          field.value
+                                            ? undefined
+                                            : 'text-muted-foreground'
+                                        )}
+                                      >
+                                        <SelectValue
+                                          ref={field.ref}
+                                          placeholder="Select a destination ..."
+                                        />
+                                      </SelectTrigger>
+                                      <SelectContent>
+                                        <ConnectionSelectContent
+                                          postgres={postgres}
+                                          mysql={mysql}
+                                          s3={s3}
+                                          mongodb={mongodb}
+                                          gcpcs={gcpcs}
+                                          dynamodb={dynamodb}
+                                          mssql={mssql}
+                                          newConnectionValue={
+                                            NEW_CONNECTION_VALUE
+                                          }
+                                        />
+                                      </SelectContent>
+                                    </Select>
+                                    <div className="relative pb-4">
+                                      {form.getValues(
+                                        `destinations.${index}.connectionId`
+                                      ) &&
+                                        destinationValidation[
+                                          form.getValues(
+                                            `destinations.${index}.connectionId`
+                                          )
+                                        ]?.isValidating && (
+                                          <Spinner className="text-black dark:text-white absolute" />
+                                        )}
+                                    </div>
+                                  </div>
                                 )}
                               </FormControl>
+                              <div>
```

Comment:
I think if you add a `flex` class, this might look a little bit nicer as it will only take up the space it needs.
<img width="596" alt="image" src="https://github.com/user-attachments/assets/9f99bd3f-1997-483b-aa42-8a45b758f3bc">

Also, it looks like the corners are rounded on the success state but not on the other state. Maybe that could be updated for consistency.

Reply:
One thing we might want to consider too is that for ephemeral environments this will always be red. But maybe that's ok.

Reply:
yeah i think that's okay, also why i didn't make it invalidate the form and the ability to proceed. but yes the flex class sounds good!

Reply:
updated :) wish tailwind could pick up `rrounded-xl` or i just need to learn how to spell...